web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js

```diff
@@ -434,12 +434,17 @@ odoo.define('web_widget_x2many_2d_matrix.X2Many2dMatrixRenderer', function (requ
                 index = widgets.indexOf(event.target),
                 first = index === 0,
                 last = index === widgets.length - 1,
-                move = 0;
+                move = 0,
+                current_row = Math.floor(index / this.columns.length);
             // Guess if we have to move the focus
-            if (event.data.direction === "next" && !last) {
+            if (["next", "next_line", "right"].indexOf(event.data.direction) > -1 && !last) {
```

Comment:
You can use this little JS trick:

```suggestion
            if (~["next", "next_line", "right"].indexOf(event.data.direction) && !last) {
```

It makes `.indexOf()` behave the same as `.includes()` for ES5 codebases (such as odoo v12).

Reply:
Great didn't know, thanks for the trick!